components/Header/_header.scss

```diff
@@ -403,8 +413,8 @@ $after-link-padding: govuk-spacing(4);
 
 .app-navigation-header__popular-link {
   display: inline-block;
-  font-size: 16px;
-  font-size: govuk-px-to-rem(16px);
+  @include govuk-font($size: 16, $weight: bold);
+  // font-size: govuk-px-to-rem(16px);
```

Comment:
can we remove the commented out font-size?

Reply:
Removed